lib/list.g

```diff
@@ -881,6 +881,34 @@ DeclareSynonym( "IntersectBlist", INTER_BLIST );
 ##
 DeclareSynonym( "SubtractBlist", SUBTR_BLIST );
 
+#############################################################################
+##
+#F  MeetBlist( <blist1>, <blist2> )
+##
+##  <#GAPDoc Label="MeetBlist">
+##  <ManSection>
+##  <Func Name="MeetBlist" Arg='blist1, blist2'/>
+##
+##  <Description>
+##  Returns <K>true</K> if <A>blist1</A> and <A>blist2</A> have <K>true</K> in the same
+##  position and <K>false</K> otherwise. It is equivalent to, but faster than
```

Comment:
I honestly don't understand this description. Which position? All of them, any? Or are the lists supposed to have length 1? (I should read the code to find out and suggest something better... but perhaps you have an idea already)

Reply:
OK it seems you copied this from the kernel comment for `FuncMEET_BLIST`. Still, it's not good. I'll suggest an alternative.